### PR TITLE
doc: refman: Fixed a TODO regarding iterators

### PR DESCRIPTION
**Pull request by davidcok**
Fixes a TODO regarding iterators

<small>By submitting this pull request, I confirm that my contribution is made under the terms of the [MIT license](https://github.com/dafny-lang/dafny/blob/master/LICENSE.txt).</small>

### CODE REVIEW

File: docs/_config.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ baseurl: "" # the subpath of your site, e.g. /blog
 url: "https://dafny-lang.github.io/dafny" # the base hostname & protocol for your site, e.g. http://example.com
 
 # Build settings
-theme: minima
+theme: jekyll-theme-slate
```
Comment (MikaelMayer) [inline docs/_config.yml:30]: Can you please post a screenshot of the result of this theme (before/after) in the PR description?

**Review by cpitclaudel — APPROVED** [review]

**Review by cpitclaudel — CHANGES_REQUESTED** [review]
I had missed the theme change; can we do that separately?

File: docs/_config.yml (hunk 1 above)
Comment (davidcok) [reply docs/_config.yml:30]: Oops - my mistake. An experiment I did not revert. I'll fix it separately.

**Comment by robin-aws** [comment]
See also https://github.com/dafny-lang/dafny/pull/1842/files - personally I find the table helpful.

**Review by cpitclaudel — APPROVED** [review]

**Pull request merged**